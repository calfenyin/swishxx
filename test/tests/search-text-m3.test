search | | -m3 | years | 0
