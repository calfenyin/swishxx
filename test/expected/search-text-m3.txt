# results: 5
100 ./GNU_GPLv2.txt 17982 GNU_GPLv2.txt
94 ./Gutenberg_License.txt 17308 Gutenberg_License.txt
14 ./Christmas_Carol,_A.txt 162261 Christmas_Carol,_A.txt
